### ISSUE TITLE
Open new windows on "master" node

### ISSUE THREAD

**Issue by juacq97** [baskerville/bspwm#964]
I'm trying that bspwm open the windows on the left side, and move the window on the left side to right side. Something like this:

     +-----------------------+    +-----------------------+  +-----------------------+
     |                       |    |           |           |  |           |           |
     |                       |    |           |           |  |           |     2     |
     |                       |    |           |           |  |           |           |
     |          1            |    |     2     |     1     |  |     3     |-----------|
     |                       |    |     ^     |           |  |     ^     |           |
     |                       |    |           |           |  |           |     1     |
     |                       |    |           |           |  |           |           |
     +-----------------------+    +-----------------------+  +-----------------------+

I discovered that I can achieve this using the the `bspc config automatic_scheme spiral` option, but this has a strange behaviour when I close windows, the layout start to rotate and sometimes the windows take stranges sizes and everything moves. So there's any way to achieve this using the `longest_side` option, or how can I make the layout stop rotating and changing when I close windows on spiral scheme?

Thanks!

**Comment by mdutra** [comment]
There's an option to turn off rotations when closing windows:

`bspc config removal_adjustment false`

**Comment by juacq97** [comment]
I tried with `bspc config removal_adjustment false`, the layout stopped to rotate, but now the focused window goes from left to right when closing windows. There's any way to keep the focused window on the first child?

**Comment by mdutra** [comment]
> but now the focused window goes from left to right when closing windows

Did you mean when **opening** windows? If you wanna open the second window on the right you can do: `bspc config initial_polarity first_child`

**Comment by juacq97** [comment]
No, I mean when I close the windows. For example, I open 4 terminals (the new terminal opens on the left side always and focuses, as I want), then, with the focus on the last terminal, I start to close them, when I close the second terminal, the focused window jumps on the right side, and the expected behavior is that always the focused window is kept on the left side and the rest of the windows are reorganized, well, the behavior that I would like to hehe.

**Comment by mdutra** [comment]
I noticed the windows swapping theirs original positions, but not exactly how you described though.
I tested it again using the last commit (5ac163b) and I think it was fixed but I'm not sure if it's working 100%.
